pom.xml

```diff
@@ -549,6 +549,23 @@
             </exclusions>
         </dependency>
 
+        <!-- use newer ORC version -->
+        <dependency>
+            <groupId>org.apache.orc</groupId>
+            <artifactId>orc-core</artifactId>
+            <version>1.7.2</version>
```

Comment:
What version was used here before?
Does hive automatically work with newer ORC version, or is any change needed within the hive code?

We generally don't use hive code to deal with ORC, but we eg use it when creating empty bucket files.
See `io.trino.plugin.hive.HiveMetadata#createEmptyFiles`.